tests/bluetooth/bsim_bt/bsim_test_audio/test_scripts/mcs_mcc.sh

```diff
@@ -33,7 +33,7 @@ Execute ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_audio_prj_conf \
   -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} -d=1 -testid=mcs -rs=23
 
 Execute ./bs_2G4_phy_v1 -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} \
-  -D=2 -sim_length=20e6 $@
+  -D=2 -sim_length=30e6 $@
```

Comment:
Have the L2CAP errors when running mcs_mcc.sh and `media_controller.sh` been fixed as part of this refactor, or are they still present?

Reply:
Will be fixed in https://github.com/zephyrproject-rtos/zephyr/pull/42445